docs/deploying_csi_vsphere_with_rbac.md

```diff
@@ -1,9 +0,0 @@
-# CCM/CSI Zones Support (Legacy)
```

Comment:
we may want to actually keep these files to preserve the links?

Reply:
content linked here is not correct.

For example - https://github.com/kubernetes/cloud-provider-vsphere/blob/master/docs/book/tutorials/deploying_cpi_and_csi_with_multi_dc_vc_aka_zones.md

Here `vsphere.conf` contains multiple vcenter servers, vSphere CSI driver does not support multiple vCenter. CPI does support multi vcenter but driver does not. 
Keeping docs separate would make things more clear. I suggest, cloud-provider-vsphere repository should not host storageclass, or PVC yamls, and redirect user to refer docs posted here in this repository.

Another reason I am removing reference to legacy doc is, we want to keep all vSphere CSI driver related documentation here in this repository.

For topology aware volume provisioning, I will be adding a new mark down file here in this repository - `features/topology_aware_volume_provisioning.md`

Reply:
good to know